docs/src/modules/spring/pages/views.adoc

```diff
@@ -132,6 +132,7 @@ In a similar way, you can include values from the request message in the respons
 SELECT :request_id, customer_id as id, name FROM customers WHERE name = :customer_name
 ----
 
+ifdef::todo[TODO: complete with example for transformed results github.com/lightbend/kalix/issues/7884]
```

Comment:
The transformation example that are talking about in https://github.com/lightbend/kalix/issues/7884 is about taking a `User` as input but indexing a different representation, like `UserView`. 

While here we should have a more concrete example of relational projection. 

The original docs, have a paragraph about returning a wrapper for collections:
https://docs.kalix.io/java/views.html#_response_message_including_the_result

Reply:
Yep, no problem. We can extend the docs when we add the transformation example.